tools/couchdb/README.md

```diff
@@ -0,0 +1 @@
+This directory contains tools that interact directly with CouchDB.
```

Comment:
here im thinking all the import scripts

Reply:
why not something more generic like "data exporters"?

Reply:
good idea